compiler/rustc_target/src/asm/csky.rs

```diff
@@ -64,9 +66,9 @@ def_regs! {
         r20: reg = ["r20","t4"],// feature high-register
         r21: reg = ["r21","t5"],// feature high-register
         r22: reg = ["r22","t6"],// feature high-register
-        r23: reg = ["r23","t7", "fp"],// feature high-register
-        r24: reg = ["r24","t8", "sop"],// feature high-register
-        r25: reg = ["r25","t9","tp", "bsp"],// feature high-register
+        r23: reg = ["r23","t7"],// feature high-register
+        r24: reg = ["r24","t8"],// feature high-register
+        r25: reg = ["r25","t9"],// feature high-register
```

Comment:
Do you have a reference to the LLVM change corresponding to this?

Reply:
https://github.com/llvm/llvm-project/blob/f9906508bc4f05d3950e2219b4c56f6c078a61ef/llvm/lib/Target/CSKY/CSKYRegisterInfo.td

Reply:
LLVM has been always like this. I referred to outdated manuals previously